lib/kcerts/ssh.go

```diff
@@ -99,6 +99,40 @@ func (a SSHAgent) Valid() bool {
 	return err == nil
 }
 
+type AgentCert struct {
+	MD5        string
+	Principals []string
+	ValidFor   time.Duration
+}
+
+// Principals returns a map where the keys are the CA's PKS and the certs identities are the values
```

Comment:
no map is returned here! `... returns the list of certificates loaded in the agent and supported by enkit`.